test/ossl_test_endian.h

```diff
@@ -7,8 +7,8 @@
  * https://www.openssl.org/source/license.html
  */
 
-#ifndef HEADER_INTERNAL_ENDIAN_H
-# define HEADER_INTERNAL_ENDIAN_H
+#ifndef OPENSSL_INTERNAL_ENDIAN_H
+# define OPENSSL_INTERNAL_ENDIAN_H
```

Comment:
this define is not in sync with the file name.

Reply:
The file is misplaced IMO; moved it from `test/ossl_test_endian.h` to `include/internal/endian.h`